web-components/tokens/formatters/cssFigmaVariables.cjs

```diff
@@ -0,0 +1,94 @@
+const PATHS = {
+  themes: "semantic-tokens",
+  viewPorts: "constraints",
+};
+
+// TODO: extract the break points from figma
```

Comment:
could you do this TODO before we merge this PR? Otherwise I might open a new issue in the project tool and link it in this TODO that it does not get forgotten

Reply:
since julio is not working anymore, I will quickly open a new ticket for this, so it is not forgotten